test/new-e2e/system-probe/config/platforms.json

```diff
@@ -1,47 +1,47 @@
 {
     "url_base": "https://dd-agent-omnibus.s3.amazonaws.com/kernel-version-testing/rootfs",
     "x86_64": {
-        "ubuntu_16.04": "master/ubuntu-16-x86_64.qcow2.xz",
-        "ubuntu_18.04": "master/ubuntu-18-x86_64.qcow2.xz",
-        "ubuntu_20.04": "master/ubuntu-20-x86_64.qcow2.xz",
-        "ubuntu_22.04": "master/ubuntu-22-x86_64.qcow2.xz",
-        "ubuntu_23.10": "master/ubuntu-23-x86_64.qcow2.xz",
-        "amzn_4.14": "master/amzn2-kvm-2.0-x86_64-4.14.qcow2.xz",
-        "amzn_5.4": "master/amzn2-kvm-2.0-x86_64-5.4.qcow2.xz",
-        "amzn_5.10": "master/amzn2-kvm-2.0-x86_64-5.10.qcow2.xz",
-        "amzn_2023": "master/amzn2023-kvm-2.0-amd64-al2023.qcow2.xz",
-        "fedora_37": "master/Fedora-Cloud-Base-37.x86_64.qcow2.xz",
-        "fedora_38": "master/Fedora-Cloud-Base-38.x86_64.qcow2.xz",
+        "ubuntu_16.04": "usama.saqib/kmt-prepare/ubuntu-16-x86_64.qcow2.xz",
```

Comment:
How are we going to handle coordinating the ami-builder changes and the agent changes without breaking anything?

Reply:
I am putting off merging this in till the versioning PRs are merged. That would allow us to merge in ami-builder first and then pull in the required version with this PR.